web/packages/api/src/assets_v2.ts

```diff
@@ -5,7 +5,7 @@ import { SnowbridgeEnvironment } from "./environment"
 import { Context } from "./index"
 import {
     buildParachainERC20ReceivedXcmOnDestination,
-    DOT_LOCATION,
+    DOT_LOCATION, dotLocationOnKusamaAssetHubLocation,
```

Comment:
The formatting doesn’t seem correct here. I’d suggest running `pnpm format` to format the code before committing.

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/1463/commits/98d2efd9cdcb350d14ce2f2c6e342d8d25620bff.